src/client.ts

```diff
@@ -120,6 +120,11 @@ export default function connect(
   dsn?: string | ConnectConfig | null,
   options?: ConnectConfig | null
 ): Promise<Connection> {
+  // tslint:disable-next-line: no-console
+  console.warn(
+    "The `connect()` function is deprecated and is scheduled to be " +
+      "removed. Use `createPool()` instead."
+  );
```

Comment:
Yeah, to be clear we'll actually deprecate the `createPool` function in the future, leaving only `connect()`. @elprans

Reply:
1. should not
2. should not

this PR *should only* deprecate `Pool.acquire` and `Pool.release`. Same in edgedb-python. Everything else is a bit too complex to do right now.

Reply:
changed. 

I also deprecated `Pool.run()` because it seemed very similar to `acquire()`.